src/log.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BENCHMARK_LOG_H_
+#define BENCHMARK_LOG_H_
+
+#include <ostream>
+
+namespace benchmark {
+namespace internal {
+
+int GetLogLevel();
```

Comment:
i did some work to get glog to use enums but it never landed. it's safer in that it's easy to check for expected levels. i suppose we don't really need that here as we're only emulating VLOG.